src/debug.c

```diff
@@ -1306,9 +1306,9 @@ void logRegisters(ucontext_t *uc) {
         serverLog(LL_WARNING, "  Dumping of registers not supported for this OS/arch"); \
     } while (0)
 
-/* OSX */
+/* Mac OS X */
 #if defined(__APPLE__) && defined(MAC_OS_10_6_DETECTED)
-    /* OSX AMD64 */
+    /* Mac OS X AMD64 */
```

Comment:
AMD64 was present as of 10.4, and the macro checks for 10.6, so the comment should use the 10.6 era brand